application/testing/CMakeLists.txt

```diff
@@ -470,6 +470,12 @@ f3d_test(NAME TestInteractionAnimationFrameRate DATA f3d.glb ARGS --animation-pr
 f3d_test(NAME TestInteractionAnimationCameraMovement DATA CameraAnimated.glb ARGS --camera-index=0 --animation-progress INTERACTION)#Space;MouseMovement;Space;
 f3d_test(NAME TestInteractionAnimationInvert DATA f3d.glb ARGS --animation-speed-factor=-1 --animation-progress INTERACTION)#Space;Wait;Space;
 
+# Test scrolling cheatsheet with no model by mouse wheel and by using scrollbar
+if(NOT APPLE)
```

Comment:
I'm afraid that's not what I meant. This test seens to work correctly on macos arm and I suppose it works fine an actual apple system. You can disable the test in `.github/actions/generic-ci/action.yml `